explainaboard_client/cli/evaluate_system.py

```diff
@@ -53,13 +34,13 @@ def main():
         help="Name of the system that you are evaluating",
     )
     parser.add_argument(
-        "--system_output",
+        "--system_output_file",
```

Comment:
I think these changes in this PR will not break the implementation of the one in the teaching class.

Reply:
I don't think it will, but I'd actually suggest that you set a specific version that you use for the class to prevent any future changes from causing issues. That's what I did as well: https://github.com/neubig/minbert-assignment/blob/main/setup.sh#L13

Reply:
>"suggest that you set a specific version that you use for the class to prevent any future changes from causing issues"

yeah, this makes sense